### PR TITLE
Fix glyphData (again)

### PR DESCRIPTION
**Pull request by khaledhosny**
Apparently I’m the only one who uses this and it keeps breaking, so this time make sure it gets tested.

### CODE REVIEW

**Comment by khaledhosny** [comment]
Any idea why all these unrelated tests are failing?

File: Lib/gftools/builder/recipeproviders/googlefonts.py [hunk 1]
```diff
@@ -221,10 +221,6 @@ def build_a_static(self, source, instance, output):
         target = os.path.join(outdir, f"{instancebase}.{output}")
 
         static_args = ""
-        if source.is_glyphs:
-            for gd in self.config.get("glyphData", []):
-                static_args += " --glyph-data " + gd
```
Comment (simoncozens) [inline Lib/gftools/builder/recipeproviders/googlefonts.py:226]: Why do we want to remove this for statics?

**Comment by m4rc1e** [comment]
> Any idea why all these unrelated tests are failing?

@khaledhosny one of our test servers is currrently experiencing issues. We'll ignore this for the time being.

**Comment by simoncozens** [comment]
The refactoring I did in #906 will clobber this.

**Comment by khaledhosny** [comment]
> The refactoring I did in #906 will clobber this.

I rebased it on current master and kept only the test. Building variable fonts is fixed, but building statics is still broken.

**Comment by khaledhosny** [comment]
> > The refactoring I did in #906 will clobber this.
> 
> I rebased it on current master and kept only the test. Building variable fonts is fixed, but building statics is still broken.

This should be fixed now. The tests now actually check that glyphData is being applied, not only that the build passes.

**Comment by khaledhosny** [comment]
The test failures does not seem to be related to this PR.

**Pull request merged**